### PR TITLE
Change the way release_ppa.sh treats branches.

### PR DESCRIPTION
**Pull request by christianparpart**
Only release branch is treated release branch,
and everything else is treated a development branch.

Implements #6899

### CODE REVIEW

File: scripts/release_ppa.sh [hunk 1]
```diff
@@ -73,11 +73,11 @@ then
     SMTDEPENDENCY=""
```
Comment (chriseth) [inline scripts/release_ppa.sh:77]: Line 58: should check for `== release`?

Comment (chriseth) [inline scripts/release_ppa.sh:77]: Please update the documentation at the start of this file, and, if needed the release checklist.

**Comment by chriseth** [comment]
Maybe better: If input is equal to `release` or starts with `v\d`

**Comment by christianparpart** [comment]
@chriseth: all comments applied.

File: scripts/release_ppa.sh [hunk 2]
```diff
@@ -3,11 +3,11 @@
 ## This is used to package .deb packages and upload them to the launchpad
 ## ppa servers for building.
 ##
-## If no argument is given, creates a package for the develop branch
-## and uploads it to the ethereum/ethereum-dev ppa.
+## You can pass an branch name as argument to this script (which, if no argument is given,
```
Comment (erak) [inline scripts/release_ppa.sh]:
```suggestion
## You can pass a branch name as argument to this script (which, if no argument is given,
```

File: scripts/release_ppa.sh [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
+## will default to "develop").
 ##
-## If an argument is given, it is used as a tag and the resulting package
-## is uploaded to the ethereum/ethereum ppa.
+## If the gien branch is "release", the resulting package will be uplaoded to
+## ethereum/ethereum PPA, or ethereum/ethereum-dev PPA otherwise.
 ##
 ## The gnupg key for "builds@ethereum.org" has to be present in order to sign
```
Comment (erak) [inline scripts/release_ppa.sh:12]: Side-note: we should document this in our release check list.

**Review by erak — APPROVED** [review]

File: scripts/release_ppa.sh (hunk 3 above)
Comment (christianparpart) [reply scripts/release_ppa.sh:12]: The release checklist is saying that you must call `release_ppa.sh release` already. I didn't see a reason to document anything else here. (maybe, that _now_ you can also write `release_ppa.sh v0.6.0` iff `v0.6.0` is your release branch)

File: scripts/release_ppa.sh [hunk 4]
```diff
@@ -3,11 +3,11 @@
...
 ## ppa servers for building.
 ##
-## If no argument is given, creates a package for the develop branch
-## and uploads it to the ethereum/ethereum-dev ppa.
+## You can pass a branch name as argument to this script (which, if no argument is given,
+## will default to "develop").
 ##
-## If an argument is given, it is used as a tag and the resulting package
-## is uploaded to the ethereum/ethereum ppa.
+## If the gien branch is "release", the resulting package will be uplaoded to
```
Comment (ekpyron) [inline scripts/release_ppa.sh:9]:
```suggestion
## If the given branch is "release", the resulting package will be uploaded to
```
;-)

**Pull request merged**